be/src/vec/runtime/vdata_stream_recvr.cpp

```diff
@@ -427,11 +428,13 @@ Status VDataStreamRecvr::create_merger(const VExprContextSPtrs& ordering_expr,
 Status VDataStreamRecvr::add_block(const PBlock& pblock, int sender_id, int be_number,
                                    int64_t packet_seq, ::google::protobuf::Closure** done) {
     SCOPED_ATTACH_TASK_WITH_ID(_query_mem_tracker, _query_id, _fragment_instance_id);
+    _mem_tracker->consume(pblock.ByteSizeLong());
```

Comment:
do not consume PB object's size, because you do not rellease its size.
And I think it is useless, because we do not save the PB in the queue.